maven-plugin/src/main/java/org/openmrs/maven/plugins/BuildDistro.java

```diff
@@ -190,7 +194,7 @@ private void deleteDistroFiles(File targetDir) {
 			new File(targetDir, OPENMRS_DISTRO_PROPERTIES).delete();
 		}
 		catch (IOException e) {
-			e.printStackTrace();
```

Comment:
I used `log.error(e.toString());` instead of  `e.printStackTrace();`.
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-HowToLog